fe4u-boilerplate-main/src/css/app.css

```diff
@@ -1,5 +1,3 @@
-/* Import fonts */
 @import url('../fonts/Roboto.css');
```

Comment:
```
@import url('../Roboto.css');
```
All CSS files should be in one folder. The folder `fonts` is for fonts files, if you download it instead of using Google Fonts.